src/dune_rules/coq/coq_lib.ml

```diff
@@ -405,6 +619,281 @@ module DB = struct
       Resolve.Memo.List.map theories ~f:(resolve ~coq_lang_version db)
     in
     Resolve.O.(theories >>= top_closure)
+
+  let empty_db : t =
+    let resolve _ = Resolve_result.Not_found in
+    { parent = None; resolve; boot_id = None }
+
+  (* We recursively descend into a directory structure and pick out directories
+     that contain a .cmxs file in order to find plugins. We rely entirely on
+     heuristics when finding plugins. (Unfortunately) *)
```

Comment:
We should make this code for deletion in the 1.1 version of the language (where we will force all plugins to go by findlib / dune)

Reply:
Let's do that when we do a 1.0. At the moment, I am unsure if that plan is sound, but that is independent from this PR.